### PR TITLE
Base Automation: Header, base menu to access to the modules

### PR DESCRIPTION
**Pull request by garyver86**

### CODE REVIEW

**Comment by garyver86** [comment]
Implements: HeaderWithLogin, HeaderWithoutLogin, calls to another pages from Login, setImputFieldDate() in CommonEvents

File: src/main/java/org/umssdiplo/automationv01/core/managepage/SHLogin.java [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+    @FindBy(name = "emp_button")
+    private WebElement employeeTab;
+
+    @FindBy(name = "accidents_button")
+    private WebElement accidentsTab;
+
+    @FindBy(name = "aassign_button")
+    private WebElement assignTab;
+
+
```
Comment (marcelogaray) [inline src/main/java/org/umssdiplo/automationv01/core/managepage/SHLogin.java]: remove blank line.

File: src/main/java/org/umssdiplo/automationv01/core/utils/CommonEvents.java [hunk 2]
```diff
@@ -19,12 +18,23 @@
...
         webElement.sendKeys(content);
     }
 
+    /**
+     * This method set text content to date web element.
+     *
+     * @param webElement Is web element.
+     * @param content    Is the content that will be set to the web element.
+     */
+    public static void setInputFieldDate(WebElement webElement, String content) {
```
Comment (marcelogaray) [inline src/main/java/org/umssdiplo/automationv01/core/utils/CommonEvents.java]: use setDateInputField

File: src/main/java/org/umssdiplo/automationv01/core/utils/PropertyAccessor.java [hunk 3]
```diff
@@ -53,15 +53,11 @@ public String getBrowser() {
...
+    public String getBaseUrl() { return getDataProperty(BASE_URL); }
 
     public String getUser() {
         return getDataProperty(USER);
     }
 
-    public String getPassword() {
-        return getDataProperty(PASSWORD);
-    }
+    public String getPassword() { return getDataProperty(PASSWORD); }
```
Comment (marcelogaray) [inline src/main/java/org/umssdiplo/automationv01/core/utils/PropertyAccessor.java]: you can format your code before sent to code review
IDE helps us on that use the following command lines:
- ctrl + alt + L
- ctrl + alt + o

This method was defined in just one line.

File: src/test/java/org/umssdiplo/automationv01/CucumberRunner.java [hunk 4]
```diff
@@ -1,14 +0,0 @@
...
-import cucumber.api.testng.AbstractTestNGCucumberTests;
-
-@CucumberOptions(
-        features = {"src/test/resources/features"},
-        format = {"pretty",
-                "html:build/cucumber",
-                "json:build/cucumber/cucumber.json",
-                "junit:build/test-report.xml"}
-)
-public class CucumberRunner extends AbstractTestNGCucumberTests{
```
Comment (marcelogaray) [inline src/test/java/org/umssdiplo/automationv01/CucumberRunner.java]: you dont need to remove/rename base core framework.

File: gradle.properties [hunk 5]
```diff
@@ -1,17 +1,13 @@
 #CHROME or FIREFOX
 browser=CHROME
 
-#features
-featureFolders =
```
Comment (marcelogaray) [inline gradle.properties]: it has a purpose that I will need to teach you (all the class). 
Dont remove it.

File: gradle.properties [hunk 6]
```diff
@@ -1,17 +1,13 @@
...
-cucumberOptions =
```
Comment (marcelogaray) [inline gradle.properties]: it has a purpose that I will need to teach you (all the class). 
Dont remove it.

File: src/main/java/org/umssdiplo/automationv01/core/managepage/SHLogin.java [hunk 7]
```diff
@@ -0,0 +1,39 @@
...
+    public SHOrganization clickOrganizationTab(){
+        CommonEvents.clickButton(organizationTab);
+        return new SHOrganization();
+    }
+
+    public SHEmployee clickEmployeeTab(){
+        CommonEvents.clickButton(employeeTab);
+        return new SHEmployee();
+    }
+    public SHAccident clickAccidentTab(){
```
Comment (marcelogaray) [inline src/main/java/org/umssdiplo/automationv01/core/managepage/SHLogin.java]: add one line between methods.

File: src/main/java/org/umssdiplo/automationv01/core/managepage/SHLogin.java [hunk 8]
```diff
@@ -0,0 +1,39 @@
...
+        CommonEvents.clickButton(employeeTab);
+        return new SHAccident();
+    }
+    public SHAssignation clickAssignTab(){
```
Comment (marcelogaray) [inline src/main/java/org/umssdiplo/automationv01/core/managepage/SHLogin.java]: add one line between methods.

**Review by marcelogaray — CHANGES_REQUESTED** [review]
review observations please.

**Comment by garyver86** [comment]
Observaciones corregidas.

**Pull request merged**